public/app/features/plugins/admin/helpers.ts

```diff
@@ -218,6 +219,7 @@ export function mapToCatalogPlugin(local?: LocalPlugin, remote?: RemotePlugin, e
     accessControl: local?.accessControl,
     angularDetected: local?.angularDetected || remote?.angularDetected,
     isFullyInstalled: Boolean(local) || isDisabled,
+    iam: local?.iam,
```

Comment:
This function munges the local installed plugin data (that comes from backend) with the remote plugin data (that comes from gcom api). Here we have the opportunity to fall back from local to remote when a plugin is or isn't installed. If my memory serves me correctly this, currently, will prevent rendering iam information for any uninstalled plugins. Is that intended?

Reply:
Changed as requested.

Reply:
I shouldn't have changed this. We had a conversation about this [[link](https://github.com/grafana/grafana/pull/78355/commits/7dcffb833186f67f85370bf1b6a5c517c96efb52#r1422420663)]

This has been reverted